### PR TITLE
Carlosjuarez password forgotten display

### PR DESCRIPTION
**Pull request by carlosjuarez**
UI Created for the password update

### CODE REVIEW

File: app/src/main/res/layout/activity_update_password.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        android:paddingEnd="@dimen/activity_horizontal_margin"
+        android:paddingStart="@dimen/activity_horizontal_margin"
+        android:paddingTop="@dimen/activity_vertical_margin"
+        app:layout_behavior="@string/appbar_scrolling_view_behavior">
+
+        <android.support.design.widget.TextInputLayout
+            android:id="@+id/til_updatepassword_password"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="8dp"
```
Comment (itzcoatl90) [inline app/src/main/res/layout/activity_update_password.xml]: Nice tried... not hardcoding...

File: app/src/main/res/layout/activity_update_password.xml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            android:layout_marginEnd="@dimen/input_margin_start_to_end"
+            android:layout_marginStart="@dimen/input_margin_start_to_end"
+            android:layout_marginTop="8dp"
```
Comment (itzcoatl90) [inline app/src/main/res/layout/activity_update_password.xml]: Nop... no hardcoding here

File: app/src/main/res/layout/activity_update_password.xml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                android:hint="@string/hint_password"
+                android:inputType="textPassword"
+                android:singleLine="true" />
+        </android.support.design.widget.TextInputLayout>
+
+        <android.support.design.widget.TextInputLayout
+            android:id="@+id/til_updatepassword_confirmpassword"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="8dp"
```
Comment (itzcoatl90) [inline app/src/main/res/layout/activity_update_password.xml]: nop

File: app/src/main/res/layout/activity_update_password.xml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+            android:layout_marginEnd="@dimen/input_margin_start_to_end"
+            android:layout_marginStart="@dimen/input_margin_start_to_end"
+            android:layout_marginTop="8dp"
```
Comment (itzcoatl90) [inline app/src/main/res/layout/activity_update_password.xml]: nop

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">FInalProject</string>
+    <string name="hint_password">Password</string>
+    <string name="hint_confirm_password">Confirmar password</string>
```
Comment (itzcoatl90) [inline app/src/main/res/values/strings.xml]: Confirmar???? What's does that means? lol

File: app/src/main/res/values/styles.xml [hunk 6]
```diff
@@ -1,11 +1,19 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
```
Comment (itzcoatl90) [inline app/src/main/res/values/styles.xml:4]: Need to coordinate with everybody this.... otherwise, you will change this in every PR

**Review by itzcoatl90 — CHANGES_REQUESTED** [review]
please address comments

**Review by itzcoatl90 — CHANGES_REQUESTED** [review]
No more comments... please solve merging conflicts

File: app/src/main/res/values/styles.xml (hunk 6 above)
Comment (carlosjuarez) [reply app/src/main/res/values/styles.xml:4]: NoActionBar is the right one,

everybody should use this file

Comment (carlosjuarez) [reply app/src/main/res/values/styles.xml:4]: I don't know how to do that, because it doesn't show a conflict in my side

**Comment by itzcoatl90** [comment]
Neither in my side.... lol.... the conflict is here ^

screen shot 2018-07-20 at 12 35 18 am

**Pull request closed without merging**